frontend/src/global-styles.jsx

```diff
@@ -1,4 +1,4 @@
-import { createGlobalStyle } from "styled-components";
+import { createGlobalStyle } from 'styled-components';
```

Comment:
Maybe i am missing something, i thought this was removed

Reply:
It was never removed.